apple/internal/docc.bzl

```diff
@@ -225,6 +228,10 @@ Must be one of "error", "warning", "information", or "hint"
                 doc = "A fallback display name if no value is provided in the documentation bundle's Info.plist file.",
                 mandatory = True,
             ),
+            "hosting_base_path": attr.string(
+                doc = "The base path your documentation website will be hosted at. For example, to deploy your site to 'example.com/my_name/my_project/documentation' instead of 'example.com/documentation', pass '/my_name/my_project' as the base path..",
```

Comment:
```suggestion
                doc = "The base path your documentation website will be hosted at. For example, to deploy your site to 'example.com/my_name/my_project/documentation' instead of 'example.com/documentation', pass '/my_name/my_project' as the base path.",
```